libbeat/docs/shared-autodiscover.asciidoc

```diff
@@ -119,35 +114,52 @@ contain variables from the autodiscover event. They can be accessed under data n
 
 include::../../{beatname_lc}/docs/autodiscover-kubernetes-config.asciidoc[]
 
+[[configuration-autodiscover-hints]]
+=== Hints based autodiscover
+
+include::../../{beatname_lc}/docs/autodiscover-hints.asciidoc[]
+
+[[configuration-autodiscover-advanced]]
+=== Advanced usage
+
+////
```

Comment:
Leftover comment?

Reply:
I left this commented as I expect new builders coming soon, they will be documented there (including hints) once we have more than just 1.